# Cartesian linear/angular velocities (Gripper)
max_vx: 0.5 # m/s
max_vy: 0.5 # m/s
max_vz: 0.5 # m/s
max_wx: 0.5 # rad/s
max_wy: 0.5 # rad/s
max_wz: 0.5 # rad/s

# Cartesian linear/angular positions (Gripper)
max_dx: 0.05  # m
max_dy: 0.065 # m
max_dz: 0.065 # m
max_dR: 0.8   # rad (~45 deg)
max_dP: 0.8   # rad (~45 deg)
max_dY: 0.8   # rad (~45 deg)

# Sign for linear/angular velocities (Gripper)
sign_vx: -1.0
sign_vy: -1.0
sign_vz: 1.0
sign_wx: 1.0
sign_wy: 1.0
sign_wz: 1.0

# Input axes (Joystick)
axis_vx: 2
axis_vy: 0
axis_vz: 1
axis_wx: 0
axis_wy: 2
axis_wz: 1

# Input buttons (Joystick)
pos_button: 0
ang_button: 3
deadman_button: 2
gripper_button: 1

# Timeout for joystick reset (<0 no timeout)
joy_msg_timeout: -1
